sympy/stats/drv_types.py

```diff
@@ -43,7 +43,8 @@ class GeometricDistribution(SingleDiscreteDistribution):
 
     @staticmethod
     def check(p):
-        _value_check(And(0 < p, p <= 1), "p must be between 0 and 1")
```

Comment:
I had to look up `_value_check` because the action is not obvious from the name. How rename it to `_whenFalse(boolean, msg)`. And why did you remove the `And`? If either is False the `And` will be False.

Reply:
We can import `_value_check` as `_whenFalse` wherever necessary. Renaming can cause a lot of efforts to be made to make changes across the stats module. I think it will be less useful and after all this depicts more of a private function due to `_` in front of it  and hence, only developers have to understand it's purpose which I believe is we can interpret from its few lines of code in `sympy.stats.rv`. What do you say?

Reply:
> I had to look up `_value_check` because the action is not obvious from the name. How rename it to `_whenFalse(boolean, msg)`.

I agree, I had to do the same. `_whenFalse` would be a better name.